registry/registry.go

```diff
@@ -130,9 +131,23 @@ func (registry *Registry) ListenAndServe() error {
 			},
 		}
 
-		tlsConf.Certificates[0], err = tls.LoadX509KeyPair(config.HTTP.TLS.Certificate, config.HTTP.TLS.Key)
-		if err != nil {
-			return err
+		if config.HTTP.TLS.LetsEncrypt.CacheFile != "" {
```

Comment:
It seems like if you specify both a TLS certificate and a LetsEncrypt cachefile, the LetsEncrypt  configuration will override the certificate. Perhaps these options should be mutually exclusive.